src/server/set_family.cc

```diff
@@ -1627,7 +1493,7 @@ bool SetFamily::ConvertToStrSet(const intset* is, size_t expected_len, robj* des
   char buf[32];
   int ii = 0;
 
-  if (GetFlag(FLAGS_use_set2)) {
+  if (true) {
```

Comment:
??

Reply:
Otherwise diffs become annoying due to indentation shifts.